js/views/userPage/UserPage.js

```diff
@@ -65,25 +78,34 @@ export default class extends BaseVw {
   }
 
   selectTab(targ) {
-    let tabViewName = targ.data('tab');
-    let tabView = this.tabViewCache[tabViewName];
-    const tabViewType = tabViewName; // the original view name is passed in for the Follow view
-
-    this.$tabTitle.text(tabViewName);
+    // if an invalid targ is passed in, set it to Store
+    if (!this.tabViews[targ] && targ !== 'Following' && targ !== 'Followers') {
+      targ = 'Store'; // eslint-disable-line no-param-reassign
```

Comment:
I know it's a pain, but we can't keep just disabling a rule. What the rule wants you to do in this case, is to assign it to a function level var and then modify the var, e.g:

```
let tabTarg = targ;  // modify tabTarg at will
```

Otherwise, we should just disable this rule in its entirety if we're just going to override it everywhere. I'm a little hesitant to do that, since the rule is valued by some and apparently mutating function arguments is a bad practice that can lead to unwanted consequences. (but i have read at least one blog where the dude disabled it).

But, let's choose a path here.